app/graphql/types/query_type.rb

```diff
@@ -5,18 +5,30 @@ class QueryType < Types::BaseObject
     field :providers,
           Types::Connections::ProviderConnection,
           connection: true,
-          max_page_size: 50,
+          max_page_size: 6,
           null: false do
       argument :filters, Types::ProviderFilter, required: false
       argument :order_by, Types::ProviderOrder, required: false
     end
     def providers(filters: nil, order_by: nil)
       filters = filters.to_h
       updated_within_days = filters.delete(:updated_within_days)
+      active_requests = filters.delete(:active_requests)
+      city = filters.delete(:city)
       scope = ::Provider.where(where(filters)).order(order(order_by))
+
       if updated_within_days
         scope = scope.where("updated_at > ?", updated_within_days.days.ago)
       end
+
+      if active_requests
+        scope = scope.where("requests @> ?", '[{"satisfied": false}]')
```

Comment:
This is fine for now— and a good trick!— but we may want to add a cache column on Provider that we maintain if this gets slow.